core/storage/user/gae_models.py

```diff
@@ -96,6 +96,11 @@ class UserEmailPreferencesModel(base_models.BaseModel):
     # None if the user has never set a preference.
     site_updates = ndb.BooleanProperty(indexed=True)
 
+    # The user's preference for receiving email when user is added as a member
+    # in exploration. This is set to True when user has never set a preference.
+    membership_notifications = ndb.BooleanProperty(
```

Comment:
Hmm well if it is like that then no problem. Since whatever we name it, it must be able to clarify why it is there. I do not have problem with it. Will do the change in next commit.

Thanx